client/client.cpp

```diff
@@ -30,7 +30,8 @@ class ClientHandler {
 	ClientHandler(std::string path) {
 		seed_data.open(path);
 		if (!seed_data.is_open()) {
-        std::cerr << "File failed to open." << std::endl;
+			std::string err = "Invalid path to seed data";
+    	throw std::invalid_argument(err);
```

Comment:
Does it not let you construct the error string directly inside?
i.e. `std::invalid_argument("Invalid path to seed data")`